api/call/session/killall.toffee

```diff
@@ -5,7 +5,7 @@
           method: 'POST'
           login_needed: true
           suffix: """
-            <h4>Cull All Sessions</h4>
```

Comment:
I think it's possible the original author really meant cull here, since it's a synonym. It's not too important, though, since the edit is also fine.

Reply:
Given that the method is kill, not cull, I'm going to guess it's probably intended to be kill.